docs/TELEMETRY.md

```diff
@@ -30,21 +28,21 @@ There is one ping per page visit, fired on the `unload` event.
 
 These are the `payload` fields from the `shield-study-addon` bucket.
 
-```
+```js
 telemetry: {
-  "version":3,
-  "study_name":"cookie-restrictions@shield.mozilla.org",
-  "branch":"CoookiesBlocked",
-  "addon_version":"2.0.0",
-  "shield_version":"5.0.3",
-  "type":"shield-study-addon",
+  "version": 3,
+  "study_name": "cookie-restrictions@shield.mozilla.org",
+  "branch": "CookiesBlocked",
```

Comment:
This was the typo fix. The rest of the changes below are just adding a space after the `:` for 3% improved readability.